groups.go

```diff
@@ -68,7 +68,7 @@ type Group struct {
 	MentionsDisabled        bool                       `json:"mentions_disabled"`
 	RunnersToken            string                     `json:"runners_token"`
 	SharedProjects          []*Project                 `json:"shared_projects"`
-	SharedRunnersEnabled    bool                       `json:"shared_runners_enabled"`
+	SharedRunnersSetting    SharedRunnersSettingValue  `json:"shared_runners_setting"`
```

Comment:
Guess `SharedRunnersEnabled`should also be deprecated instead of removed right?

Reply:
Hi @svanharmelen, in my understanding, `SharedRunnersEnabled` is not an attribute of `Group` resource in the first place. `SharedRunnersEnabled` is an output of `Project` resource. So, I think we should remove `SharedRunnersEnabled` here.  cc: @timofurrer

Reply:
@svanharmelen @takp it seems like I've introduced this field a while back (https://github.com/xanzy/go-gitlab/pull/1429) - wrongly 😰

Reply:
Check 👍🏻